pythonfmu/pythonfmu-export/src/pythonfmu/PySlaveInstance.cpp

```diff
@@ -65,33 +143,17 @@ PySlaveInstance::PySlaveInstance(std::string instanceName, std::string resources
             handle_py_exception("[ctor] PyObject_GetAttrString", gilState);
         }
         int success = PyList_Insert(sys_path, 0, PyUnicode_FromString(resources_.c_str()));
+        PyList_Append(sys_path, PyUnicode_DecodeFSDefault(resources_.c_str()));
```

Comment:
This is a change unrelated to this feature request

Reply:
Removed!